server/features/content_copy.feature

```diff
@@ -28,7 +28,7 @@ Feature: Copy Content in Personal Workspace
       {"state": "draft"}
       """
       When we get "/archive/#copy._id#?version=all"
-      Then we get list with 3 items
+      Then we get list with 4 items
       When we get "/archive/"
```

Comment:
Copy operation creates an extra version in the copied article in order to record the operation.